spec/features/manage_cmr/subscriptions/create_subscriptions_spec.rb

```diff
@@ -68,6 +68,12 @@
             end
           end
 
+          after do
```

Comment:
add comment for all these afters that they should be superfluous when we can reset_provider